src/__forks__/fetch.js

```diff
@@ -11,5 +11,11 @@
 
 'use strict';
 
-require('whatwg-fetch');
-module.exports = self.fetch.bind(self);
+// This hopefully supports the React Native case, which is already bringing along
+// its own fetch polyfill. That should exist on `global`. If that doesn't exist
+// then we'll try to polyfill, which might not work correctly in all environments.
+if (global.fetch) {
+  module.exports = global.fetch.bind(global);
+} else {
+  module.exports = require('isomorphic-fetch');
+}
```

Comment:
It works fine, both on web and native.

The RN packager actually still pulls in isomorphic fetch, unfortunately, because it doesn't know about conditional requires. But, there's no error like there was with whatwg-fetch.